src/app-logic/__mocks__/l10n.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+/* global jest */
+const realL10n = jest.requireActual('../l10n.js');
```

Comment:
I was having a problem here also. So changed this to relative path and it's working as expected. I was able to see the the FTL strings when I replace the ID and removing the `unmock` statement.

Reply:
ah I think I did it also locally indeed, I forgot to mention it. Good finding!

Reply:
mmm no I didn't do it, so I don't know what happens on your side. But, this does no harm either so we can keep it.